bin/yb-ctl

```diff
@@ -1342,6 +1355,18 @@ class ClusterControl:
         alive (based on valid PID), then this function returns True. Anything else will end up
         returning False.
         """
+        # Wait for master and tserver processes to be ready.
+        def call_check_for_processes(should_get_error):
+            pids = self.for_all_daemons(self.get_pid)
+            if self.args.verbose:
+                logging.info("Daemon PIDs found after starting daemons: {}".format(pids))
+            return all(sum(pids.values(), []))
```

Comment:
why do we need a sum here? isn't pids already a list, with some items being None? won't all(pids) be enough?

Reply:
`pids` is a dict with two keys and list values.  The `sum` flattens it.

Reply:
The signature for `for_all_daemons` could change to just return a flattened list, but that would be a loss of information.  One consequence is that the log message above would be less clear.